tests/pages/pocket/base.py

```diff
@@ -51,16 +51,11 @@ class Navigation(BaseRegion):
         _mobile_menu_close_btn_locator = (By.CLASS_NAME, "mobile-nav-close-btn")
         _mobile_menu_locator = (By.CLASS_NAME, "mobile-nav")
         _mobile_menu_wrapper_locator = (By.CLASS_NAME, "mobile-nav-wrapper")
-        _home_mobile_menu_locator = (By.CSS_SELECTOR, '.mobile-nav-list-link[href="https://getpocket.com/home?src=navbar"]')
-        _my_list_mobile_menu_locator = (By.CSS_SELECTOR, '.mobile-nav-list-link[href="https://getpocket.com/my-list?src=navbar"]')
+        _mobile_menu_link_locator = (By.CSS_SELECTOR, ".mobile-nav-list-link")
```

Comment:
Question: this will find the first element in a page that matches this selector (in the same way that `querySelector('.mobile-nav-list-link')` would in JS. Is that what you're expecting here?

If not, would perhaps asserting the whole list of links (`.mobile-nav-list`) is displayed instead be another option?

Reply:
Ok, feels maybe a little non-specific for a test perhaps, but I guess if we're only trying to assert _"at least one link with the selector `mobile-nav-list-link` is displayed in the nav"_, than it works for me 👍

Reply:
I don't feel too strongly about it, sounds like `mobile-nav-list` would make more sense here (more specific selector, and inclusive of all mobile nav links), will update